examples/tutorials/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>1) typo in efficent</p><p> </p><p>2) If the protocol is for : 3 parties with one dishonest party in the malicious adversary model - how is this protocol be made to work with 2 parties + 1 crypto-worker? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Well the secureNN requires an honest majority for all parties<em> that store a share</em> - that doesn't include the crypto-worker. So 2 parties + 1 crypto-worker where one party is dishonest isn't considered a honest majority which means secureNN can't safely be used as far as I understand. </p>

Reply:
Oh, interesting! Thanks for the explanation :)